lib/ansible/modules/monitoring/icinga2_host.py

```diff
@@ -0,0 +1,332 @@
+    required: true
+  force:
+    description:
+      - If C(yes) and C(dest) is not a directory, will download the file every
+        time and replace the file if the contents change. If C(no), the file
+        will only be downloaded if the destination does not exist. Generally
+        should be C(yes) only for small local files.
+      - Prior to 0.6, this module behaved as if C(yes) was the default.
+    version_added: '0.7'
+    default: 'no'
+    type: bool
+    aliases: [ thirsty ]
+  use_proxy:
+    description:
+      - if C(no), it will not use a proxy, even if one is defined in
+        an environment variable on the target hosts.
+    default: 'yes'
+    type: bool
+  validate_certs:
+    description:
+      - If C(no), SSL certificates will not be validated. This should only be used
+        on personally controlled sites using self-signed certificates.
+    default: 'yes'
+    type: bool
+  url_username:
+    description:
+      - The username for use in HTTP basic authentication.
+      - This parameter can be used without C(url_password) for sites that allow empty passwords.
+  url_password:
+    description:
+        - The password for use in HTTP basic authentication.
+        - If the C(url_username) parameter is not specified, the C(url_password) parameter will not be used.
+  force_basic_auth:
+    description:
+      - httplib2, the library used by the uri module only sends authentication information when a webservice
+        responds to an initial request with a 401 status. Since some basic auth services do not properly
+        send a 401, logins will fail. This option forces the sending of the Basic authentication header
+        upon initial request.
+    default: 'no'
+    type: bool
+  client_cert:
+    description:
+      - PEM formatted certificate chain file to be used for SSL client
+        authentication. This file can also include the key as well, and if
+        the key is included, C(client_key) is not required.
+  client_key:
+    description:
+      - PEM formatted file that contains your private key to be used for SSL
+        client authentication. If C(client_cert) contains both the certificate
+        and key, this option is not required.
+  state:
+    description:
+      - Apply feature state.
+    required: false
+    choices: [ "present", "absent" ]
+    default: present
+  name:
+    description:
+      - name used to create / delete the host
+      - this does not need to be the FQDN, but does needs to be uniuqe
+    required: true
+  zone:
+    description:
+      - the zone from where this host should be polled
+  template:
+    description:
+      - the template used to define the host
+    required: false
+    default: None
+  check_command:
+    description:
+      - the command used to check if the host is alive
+    required: false
+    default: "hostalive"
+  display_name:
+    description:
+      - the name used to display the host
+    required: false
+    default: <name>
```

Comment:
is this really "<name>" or should this point out that "value of the name param" is taken? Please make this clear here.

Reply:
Make clear (i hope)